portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,15 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+const navigate = (pageid) => document.getElementById(pageid).scrollIntoView({behavior:"smooth",block:"start"});
+
+const initialiseNavigation = () => {
+  var navigationItems = document.getElementsByClassName("navigation-item");
+  for(var i = 0; i <navigationItems.length;i++) navigationItems[i].addEventListener("click",function(){
```

Comment:
[prefer a `for - of` loop, if possible](https://google.github.io/styleguide/jsguide.html#features-for-loops)

Reply:
changed to for - of here 6bff026